lib/connect_proto/src/order.proto

```diff
@@ -112,5 +113,6 @@ message Order {
   Patient patient = 2;
   Visit visit = 3;
   Order order = 4;
-  google.protobuf.Any subject = 5;
+  Substances substances = 5;
```

Comment:
Looking through the Order.Order proto, and how _it_ contains many of the fields that we will _also_ need (at least compared to Redox's Vaccination model) like ClinicalInfo, Provider, Location, etc., I'm wondering if perhaps `substances` should _not_ be a root node but rather a child of `Order.Order`. What distinguishes between a Vaccination and a Test would be the presence of `substances` _or_ [`Procedure`](https://github.com/PrimaryDotHealth/connect-proto/blob/main/lib/connect_proto/src/order.proto#L95)? Or even, could a vaccination be considered a Procedure itself where as part of the procedure it could have `administered_substances` or something like that?

Reply:
You don't need a separate `Substances` proto btw
```suggestion
  repeated Substance substances = 5;
```
Also, what do you think of the name `administered_substances` vs just `substances`?

Reply:
`administered_substances` definitely provides more context

Reply:
I was thinking of changing to repeated after seeing results

Reply:
You know what it totally makes sense for procedure to have administered_substances, I'll build that